main.py

```diff
@@ -145,7 +145,7 @@ def main():
         for x, y in tqdm(trainloader, desc="Training"):
             x, y = x.to(device), y.to(device)
             logits = model.forward(x)
-            loss = criterion(logits, y)
+            loss = criterion(logits, y) 
```

Comment:
```suggestion
            loss = criterion(logits, y)
```

remove whitespace after `)` 🤓